change_log/next/FE-1864-changelog.yml

```diff
@@ -0,0 +1 @@
+Other: "Change next/*.yml files for v9.0.0 release to more fully describe the changes"
```

Comment:
Just wondering whether this item is actually needed in the release notes? (It seems to be a Sage-internal pre-release documentation-only task, rather than a public-facing code change.)

Reply:
Thanks for deleting `change_log/next/FE-1864-changelog.yml` in 86e0a33 :thumbsup: